bindings/wasm/src/did/wasm_document.rs

```diff
@@ -468,5 +463,5 @@ impl From<IotaDocument> for WasmDocument {
 #[wasm_bindgen]
 extern "C" {
   #[wasm_bindgen(typescript_type = "DIDUrl | string")]
-  pub type UWasmMethodQuery;
+  pub type UDIDUrlQuery;
```

Comment:
Not very related to this PR, but it feels like there should be a better name for this. Would `DIDUrlOrString` be acceptable? Or some other convention we can come up with to encode duck-typed types, like `DuckDIDUrlQuery`?

Reply:
It maps to `DIDUrlQuery` in Rust and the `U*` prefix stands for `Union`, I didn't really have a better naming convention at the time.

Reply:
Leaving this as-is since the name is internal-only and not exported to Typescript definitions; only the `DIDUrl | string` is seen by consumers of the bindings.